core/src/main/scala/aima/core/agent/Actuator.scala

```diff
@@ -5,15 +5,14 @@ import aima.core.random.Randomness
 /**
   * @author Shawn Garner
```

Comment:
@benderpremier Please add yourself below the other names as an author to anything you change in a meaningful way.